bodhi/server/models.py

```diff
@@ -2487,6 +2494,17 @@ def url(self):
         url = '/updates/' + self.update.title + '#comment-' + str(self.id)
         return url
 
+    def unique_testcase_feedback(self):
```

Comment:
Let's give this a docblock explaining what it does, and what the return type is.